package org.forcepower.starcement.bean;

/**
 * Created by amit on 09/01/2017.
 * invoice info data model
 */

public final class CashTransferReceive
{

    String cash_trans_rcv_trans_id ="";
    String despatcher_code ="";
    String receiver_code		="";
    String despatch_value		="";
    String rec_value		="";
    String status		="";
    String transaction_type		="";
    String cash_transfer_id		="";
    String cash_transfer_date_time		="";

    public void setcash_trans_rcv_trans_id(String cash_trans_rcv_trans_id)
    {
        this.cash_trans_rcv_trans_id =cash_trans_rcv_trans_id;
    }
    public String getcash_trans_rcv_trans_id()
    {
        return cash_trans_rcv_trans_id;
    }

    public void setdespatcher_code(String despatcher_code)
    {
        this.despatcher_code=despatcher_code;
    }
    public String getdespatcher_code()
    {
        return despatcher_code;
    }

    public void setreceiver_code(String receiver_code)
    {
        this.receiver_code=receiver_code;
    }
    public String getreceiver_code()
    {
        return receiver_code;
    }
    public void setdespatch_value(String despatch_value)
    {
        this.despatch_value=despatch_value;
    }
    public String getdespatch_value()
    {
        return despatch_value;
    }

    public void setrec_value(String rec_value)
    {
        this.rec_value=rec_value;
    }
    public String getrec_value()
    {
        return rec_value;
    }

    public void setstatus(String status)
    {
        this.status=status;
    }
    public String getstatus()
    {
        return status;
    }

    public void settransaction_type(String transaction_type)
    {
        this.transaction_type=transaction_type;
    }
    public String gettransaction_type()
    {
        return transaction_type;
    }

    public void setcash_transfer_id(String cash_transfer_id)
    {
        this.cash_transfer_id=cash_transfer_id;
    }
    public String getcash_transfer_id()
    {
        return cash_transfer_id;
    }

    public void setcash_transfer_date_time(String cash_transfer_date_time)
    {
        this.cash_transfer_date_time=cash_transfer_date_time;
    }
    public String getcash_transfer_date_time()
    {
        return cash_transfer_date_time;
    }
}
